modules/preview-react/information-highlight/index.tsx

```diff
@@ -0,0 +1,79 @@
+import * as React from 'react';
+import {createContainer} from '@workday/canvas-kit-react/common';
+import {createModifiers, createStyles, cssVar} from '@workday/canvas-kit-styling';
+import {base, system} from '@workday/canvas-tokens-web';
+
+import {Base, BaseProps} from './lib/Base';
+import {Heading} from './lib/Heading';
+import {Body} from './lib/Body';
+import {Icon} from './lib/Icon';
+import {Button} from './lib/Button';
+import {useInformationHighlightModel} from './lib/modelHook';
+
+type Variant = 'informational' | 'caution' | 'attention';
```

Comment:
this seems to be repeated in a few files. You are already exporting it in `modelHook` so can you just import it from there?

Reply:
yes :D